src/styles/main.scss

```diff
@@ -1,3 +1,233 @@
-body {
+$color-main-title: #060b35;
```

Comment:
checklist p.9 [BEM] - create a separate file per each BEM block styles that have the same name as the block

Reply:
done)